flows/route.go

```diff
@@ -2,8 +2,9 @@ package flows
 
 // Route describes leaving a node
 type Route struct {
-	exit  ExitUUID
-	match string
+	exit       ExitUUID
+	match      string
+	matchExtra map[string]string
```

Comment:
don't feel strongly but would support this just being `extra` :)

Reply:
I'm fine with that